app/jobs/calendar_importer/events/base.rb

```diff
@@ -15,7 +15,7 @@ def rrule
     end
 
     def sanitize_invalid_char(input)
-      input = I18n.transliterate(input)
+      # input = I18n.transliterate(input)
       input.encode('utf-8', :invalid => :replace, :undef => :replace, :replace => '')
```

Comment:
while we're here can we just kill this ancient hash syntax pls :) otherwise looks good!